kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/callbacks/PrePostUIMethods.java

```diff
@@ -0,0 +1,17 @@
+package com.applozic.mobicomkit.uiwidgets.kommunicate.callbacks;
+
+import java.io.File;
+
+/**
+ * the callback to the pre and post methods to doInBackground for a async task
+ * can be used to implement the pre post functionality according to our needs
+ * currently, will be used for the FileTaskAsync
+ * TODO: make the interface generic
+ *
+ * @author shubham
+ * 22 September, 2019
+ */
+public interface PrePostUIMethods {
+    void preTaskUIMethod();
+    void postTaskUIMethod(boolean b, File file);
```

Comment:
@shubhamtewari please use proper variable names and not a, b, c, d

Reply:
changed

Reply:
still showing boolean b

Reply:
should be changed now